src/map/unit.cpp

```diff
@@ -1815,14 +1815,16 @@ TIMER_FUNC(unit_resume_running){
 
 /**
  * Sets the delays that prevent attacks and skill usage considering the bl type
- * Officially it just remembers the last attack time here and applies the delays during the comparison
+ * Officially for players it just remembers the last attack time here and applies the delays during the comparison
```

Comment:
We should not expose the full logic into the documentation of the function.
Only caller relevant information should be in there.
For the caller the first line is enough.
The rest should be incorporated into the logic of the function itself (now we have a [partially] undocumented switch again).

Reply:
Yeah I was also wondering where to put all the information.